h/static/scripts/annotation-ui-controller.js

```diff
@@ -21,7 +21,7 @@ function AnnotationUIController($rootScope, $scope, annotationUI) {
   });
 
   $rootScope.$on(events.ANNOTATION_DELETED, function (event, annotation) {
-    annotationUI.removeSelectedAnnotation(annotation);
```

Comment:
Not your fault, but what a horrible method name. `unselect(annotation)` or `removeFromSelection(annotation)` or something would be clearer.